mvrx-navigation/build.gradle

```diff
@@ -0,0 +1,135 @@
+apply plugin: 'com.android.library'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-android-extensions'
+apply plugin: 'jacoco'
+apply plugin: 'com.vanniktech.maven.publish'
+
+android {
+    compileSdkVersion versions.compileSdk
+    buildToolsVersion versions.buildTools
+    resourcePrefix "mvrx_"
+
+    defaultConfig {
+        minSdkVersion versions.minSdk
+        targetSdkVersion versions.targetSdk
+        consumerProguardFiles "proguard-rules.pro"
+    }
+
+    lintOptions {
+        abortOnError true
+        textReport true
+        textOutput "stdout"
+    }
+
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
+
+    kotlinOptions {
+        jvmTarget = JavaVersion.VERSION_1_8.toString()
+    }
+
+    sourceSets {
+        main.java.srcDirs += 'src/main/kotlin'
+        test.java.srcDirs += 'src/test/kotlin'
+        test.res.srcDirs += 'src/test/res'
+    }
+
+    testOptions {
+        unitTests {
+            includeAndroidResources = true
+            all {
+                jacoco {
+                    includeNoLocationClasses = true
+                }
+            }
+        }
+    }
+}
+
+androidExtensions {
+    experimental = true
+}
+
+jacoco {
+    toolVersion = "0.8.2"
+}
+
+dependencies {
+    implementation project(":mvrx")
+    implementation project(":testing")
+
+    implementation libraries.lifecycleCommonJava8
+    implementation libraries.kotlin
+
+    testImplementation testLibraries.mockito
+    testImplementation testLibraries.junit
+    testImplementation testLibraries.roboeletric
+    testImplementation libraries.rxAndroid
+}
+
+// module specific dependencies
+dependencies {
+    def nav_version = "2.3.0-alpha04"
```

Comment:
I am using your branch in a personal project of mine (that is the whole reason why I was pushing the refactors 😉).

@marukami I was trying to figure out why you made two separated segments of `dependencies` into this `build.gradle`.

Reply:
> @marukami I also had hard times with alpha libraries from Google.

I've found the Androidx stuff to be fine and most issues you can workaround. Far as I can tell Stable and Alpha are mostly the same except the newer APIs, tho this is Andoidx and Google so who knows how they are composing the stable and alpha channel code. 

For the Alpha here I was planning to use the `TestNavHostController` that comes in the `2.3.0-alpha`; I've found it a lot better than mocking in a few projects. Tho, I did not end up using it, so, yer I think `2.2.1` should work.

`2.2.1` it is then.